io_scene_godot/converters/material/script_shader/shader_functions.py

```diff
@@ -245,6 +245,41 @@ def __init__(self, code, input_sockets, output_properties):
 void node_combine_rgb(float r, float g, float b, out vec4 color) {
     color = vec4(r, g, b, 1.0);
 }
+"""),
+
+	ShaderFunction(code="""
+void node_hsv(float fac, float in_hue, float in_saturation, float in_value,
```

Comment:
The added code here can be aligned better :)

Reply:
I honestly just followed the rest of the code's design, thought any additional code should look like that

Reply:
Forgive my English, what I really mean is "indentations".
From my browser, I am seeing some inconsistent indentations
![image](https://user-images.githubusercontent.com/19422835/127324089-8b1f9342-b394-4f69-bc5f-6837ca45ddd0.png)

However I just found when I copy this piece of code into my editor, they are all four spaces indented. Honestly I don't know what's happening.  One thing for sure is there are some extra spaces can be removed at line 262, line 266, and line 270

Reply:
I believe github shows Tabs as 8 spaces, while most editors show it as 4.

Reply:
> I believe github shows Tabs as 8 spaces, while most editors show it as 4.

Note that you can install the [Refined GitHub](https://github.com/sindresorhus/refined-github) browser add-on to change this :slightly_smiling_face:

Reply:
Oh okay, I know what the problem is, those extra lines were tabs, but blender expects everything to be spaces, I converted TABs into spaces on node_convertes.py but not on shader_functions.py, gonna fix it real quick